nautobot/wireless/forms.py

```diff
@@ -183,7 +183,7 @@ class RadioProfileBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):
         widget=StaticSelect2(),
     )
     channel_width = JSONArrayFormField(
-        choices=add_blank_choice(RadioProfileChannelWidthChoices),
+        choices=RadioProfileChannelWidthChoices,
```

Comment:
I fixed this issue with `JSONArrayFormField` because it's pretty "small", but I have this weird feeling that I should've create a bug with proper description and then a separate bugfix with test for that.